features/251-templates/go-i18n-research.md

```diff
@@ -0,0 +1,63 @@
+# How could one do i18n in Go
+
+This document concludes the research on how to do i18n in Go as of 2020-03.
```

Comment:
Do we have a conclusion on how to do it at the moment?

Reply:
Sorry. I should have mentioned that I tend to pick the last choice: Implement MessageFormat ourselves.